### ISSUE TITLE
VoiceOver: character should be spoken when entered

### ISSUE THREAD

**Issue by dylanhand** [divvun/giellakbd-ios#124]
To be consistent with native VoiceOver support, the character that's input by the user should be spoken by VoiceOver in a higher-pitched voice. Ours currently gives only the selection sound effect as feedback.
